lib/atscppapi/examples/websocket/WebSocket.h

```diff
@@ -38,6 +38,11 @@ using atscppapi::InterceptPlugin;
 using atscppapi::Transaction;
 using atscppapi::GlobalPlugin;
 
+namespace
+{
+GlobalPlugin *plugin;
+}
```

Comment:
That header file is only included for by this one plugin. I can move it though. I'll also fix the coverity issues that just came out.